rllib/policy/policy.py

```diff
@@ -189,6 +192,39 @@ def compute_single_action(self,
         return single_action, [s[0] for s in state_out], \
             {k: v[0] for k, v in info.items()}
 
+    @DeveloperAPI
+    def _compute_actions_from_trajectories(
```

Comment:
Can you remove.the underscore, but mark this as experimental for now instead of developerapi?

Reply:
+1

Reply:
done